packages/react-client/src/ReactFlightClient.js

```diff
@@ -722,15 +724,58 @@ function createElement(
         value: task,
       });
     }
-    // TODO: We should be freezing the element but currently, we might write into
-    // _debugInfo later. We could move it into _store which remains mutable.
-    if (initializingChunkBlockedModel !== null) {
+  }
+
+  if (initializingHandler !== null) {
+    const handler = initializingHandler;
+    // We pop the stack to the previous outer handler before leaving the Element.
+    // This is effectively the complete phase.
+    initializingHandler = handler.parent;
+    if (handler.errored) {
+      // Something errored inside this Element's props. We can turn this Element
+      // into a Lazy so that we can still render up until that Lazy is rendered.
```

Comment:
Should there be a new unit test for this? It's still a bit unclear to me how this affects siblings.

Reply:
This is effectively covered by the Throw cases that has the new component stacks. It is split into a synchronous and an asynchronous case where the error is already known vs becomes known though. Not sure if both are covered but meh.